images/ubuntu/templates/locals.ubuntu.pkr.hcl

```diff
@@ -0,0 +1,21 @@
+locals {
+  image_properties    = var.image_sku == "22_04-lts" ? {
+                            image_os = "ubuntu22"
+                            os_disk_size_gb = var.os_disk_size_gb != 0 ? var.os_disk_size_gb : 75
+                      } : {
+                            image_os = "ubuntu24"
+                            os_disk_size_gb = var.os_disk_size_gb != 0 ? var.os_disk_size_gb : 75
+                      }
+}
+
+// Define local variables for the ubuntu-minimal build
```

Comment:
This comment is relevant for full `locals` block or only first property?

Reply:
This is true for the entire block. I had to copy it from the `ubuntu-minimal` template to make the refactoring work. If we're not using the ubuntu-minimal image, maybe we should remove it?